packages/playwright-core/src/cli/cli.ts

```diff
@@ -115,6 +115,27 @@ program
     .option('--with-deps', 'install system dependencies for browsers')
     .option('--force', 'force reinstall of stable browser channels')
     .action(async function(args: string[], options: { withDeps?: boolean, force?: boolean }) {
+      const isLikelyNpxGlobal = process.argv.length >= 2 && process.argv[1].includes('_npx');
+      if (isLikelyNpxGlobal) {
+        console.error(wrapInASCIIBox([
+          `WARNING: It looks like you are running 'npx playwright install' without first`,
+          `installing your project's dependencies.`,
+          ``,
+          `To avoid unexpected behavior, please install your dependencies first, and`,
+          `then run Playwright's install command:`,
+          ``,
+          `    npm install`,
+          `    npx playwright install`,
+          ``,
+          `If your project does not yet depend on Playwright, first install the`,
+          `applicable npm package (most commonly @playwright/test), and`,
+          `then run Playwright's install command to download the browsers:`,
```

Comment:
This case is confusing or just plain wrong. It only applies for @playwright/test. If all they are after is `playwright` package, that will install the browsers for them, the the `npx playwright install` command is not needed and confusing.